stdlib/3/builtins.pyi

```diff
@@ -495,6 +495,8 @@ class memoryview(Sized, Container[int]):
     ndim = ...  # type: int
 
     def __init__(self, obj: Union[bytes, bytearray, memoryview]) -> None: ...
+    def __enter__(self) -> memoryview: ...
+    def __exit__(self, exc_type: Optional[BaseException], exc_val: Optional[BaseException], exc_tb: Optional[TracebackType]) -> bool: ...
```

Comment:
`exc_type` should be `Optional[Type[BaseException]]`.

Reply:
Wow, somehow I managed to drop the `Type` while copy-pasting and then manually deleted the extra brackets.  Fixed now!